validator/src/journal/chain.rs

```diff
@@ -851,8 +857,33 @@ impl<TEP: ExecutionPlatform + Clone + 'static, PV: PermissionVerifier + Clone +
                 );
                 warn!("{}", err_str);
                 ChainControllerError::ConsensusError(err_str)
-            })?
-            .unwrap_or_else(|| String::from(""));
+            })?;
+
+        // If engine not configured, fall back to defaults
+        let (name, version) = match (name, version) {
+            (Some(name), Some(version)) => (name, version),
+            _ => {
+                // If PoET settings exist, fall back to PoET
+                if settings_view
+                    .get_setting_str("sawtooth.consensus.poet.enclave_module_name", None)
+                    .map_err(|settings_err| {
+                        let err_str = format!(
+                            "Error getting sawtooth.consensus.poet.enclave_module_name from \
+                             settings view: {:?}",
+                            settings_err
+                        );
+                        warn!("{}", err_str);
```

Comment:
Will this be double logged since it is also propogated up?

Reply:
Yep, it seems so. I'll fix that.

Reply:
Fixed